src/client/components/TextsPage/style.css

```diff
@@ -0,0 +1,18 @@
+.section {
+    color: #FFFFFF;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.article {
+    column-count: 2;
+    column-gap: 40px;
+    width: 800px;
+
```

Comment:
Pusta linia do :wastebasket:

Reply:
Pan to masz oko. Poprawione. Ja mam pisać tutaj że coś poprawiłem ? czy jak to zrobie i synchronizuje to będziesz to widzał :D ?